app/models/polling/scheduler.rb

```diff
@@ -0,0 +1,45 @@
+require 'rufus-scheduler'
+module Polling
+  class Scheduler
+    attr_reader :rufus_scheduler
+    attr_reader :job
+    DEFAULT_TIMEOUT = '30.0s'.freeze
```

Comment:
On reflection, I think this is too short.  Currently, the config service is set to cache the certs for 30 seconds, so we need the timeout to be a bit longer than that to give us some wriggle room.  How about 40 seconds, or even a minute?